src/hec/firo/Testing.java

```diff
@@ -96,7 +118,82 @@ public void SqLiteTest() {
         {
             System.out.println("Error: "+e.getMessage());
         }
+    }
+
+    /**
+     * ensembleWriter may be used to test
+     * writing large amounts of ensemble data
+     *
+     * Results:
+     *  initial: 420 seconds to write a file 8.17 gb in size
```

Comment:
that looks good - about 7 minutes - can we optimize and make it faster?

Reply:
Compression appears to be the bottleneck.  My research shows default compression is aggressive at the expense of speed. Would you rather have a bigger file but faster?  Below is a screen shot of reading (from csv) and writing to sqlite for 60 days in a single watershed.


![image](https://user-images.githubusercontent.com/4818531/71680282-81600f80-2d3e-11ea-9be6-8998f2746a67.png)

Reply:
I have been thinking about compression - I believe that we will want to be flexible in how we compress and decompress the blob - for instance, what if we utilize line simplification to "compress" the data we will need to be able to compress and decompress it.

maybe this is a candidate for abstraction through injection or some other method.